test/e2e-go/cli/goal/expect/goalExpectCommon.exp

```diff
@@ -99,8 +99,16 @@ proc ::AlgorandGoal::StopNetwork { NETWORK_NAME TEST_ALGO_DIR TEST_ROOT_DIR } {
     set timeout 60
     puts "Stopping network: $NETWORK_NAME"
     spawn goal network stop -d $TEST_ALGO_DIR -r $TEST_ROOT_DIR
+    set NETWORK_STOP_MESSAGE "";
```

Comment:
I think that the semi-colon at the end of line is redundant. you might want to remove it to match the coding style here.